Tools/condor/batchList.py

```diff
@@ -94,13 +109,15 @@ def recSearch(path, outDict = None, eosurl = "root://cmseos.fnal.gov", match=".*
                     f.close()
             
                     if options.copy:
-                        eoscp(fileName, startPath, options.force, eosurl = options.eosurl)
+                        #eoscp(fileName, startPath, options.force, eosurl = options.eosurl)
```

Comment:
why break functionality here?  Please fix this to either work in all use cases or remove the hack